src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -21,14 +21,14 @@ internal class MonoProxy : DevToolsProxy
         private IList<string> urlSymbolServerList;
         private HashSet<SessionId> sessions = new HashSet<SessionId>();
         protected Dictionary<SessionId, ExecutionContext> contexts = new Dictionary<SessionId, ExecutionContext>();
-        private const string sPauseOnUncaught = "pause_on_uncaught";
-        private const string sPauseOnCaught = "pause_on_caught";
 
         public static HttpClient HttpClient => new HttpClient();
 
         // index of the runtime in a same JS page/process
         public int RuntimeId { get; private init; }
         public bool JustMyCode { get; private set; }
+        public PauseOnExceptionsKind PauseOnExceptions { get; set; }
```

Comment:
I think this should be private, and maybe named something like `_defaultPauseOnExceptions`, and initialized to `Unset`.